macosx/HBPresetsManager.m

```diff
@@ -296,7 +296,7 @@ - (void)selectNewDefault
     __block HBPreset *normalPreset = nil;
     __block HBPreset *firstUserPreset = nil;
     __block HBPreset *firstBuiltInPreset = nil;
-    __block BOOL defaultAlreadySetted = NO;
+    __block BOOL defaultAlreadySet = NO;
```

Comment:
Setted does technically exist as jargon in educational use. Outside of that scope, it's never the right thing, and arguably that case is educators betraying the English language for fun.